modules/popup-stack/lib/PopupStack.ts

```diff
@@ -1,3 +1,5 @@
+import screenfull from 'screenfull';
```

Comment:
[screenfull](https://github.com/sindresorhus/screenfull) was chosen because it is as lightweight as possible and already used by some teams.